guides/process/support/triage.md

```diff
@@ -0,0 +1,78 @@
+# Support Triage
+
+Triage is the usual entry point to the support process. Customers will email their request to support@madetech.com, which is routed to [Zendesk](https://madetechsupport.zendesk.com).
+
+On ocassion customers will ask for things via Slack or phone, in which case we can create the ticket on their behalf in Zendesk.
+
+## Process Overview
+
+It is the responsibility of the Triage Pair to:
+
+- [ ] 1. Triage the oldest item from the queue
+- [ ] 2. Record time in Harvest
+- [ ] 3. Ensure we're meeting our Service Level Agreements
+- [ ] 4. Ensure customers are proactively made aware when their hours are close to running out
+
+## 1. Triage request
+
+Triaging involves analysing the customer's request and figuring out the best way to deal with it. The course of action depends on the nature of the request.
+
+### Mission critical request
+
+If you reasonably believe the request to be mission critical (i.e. the customer application is offline, a major piece of functionality is unavailable), then we would typically use best efforts to restore that service, regardless of the time needed.
+
+### Expected to take less than 2 hours
+
+For less critical requests, if we expect delivering the solution to take less than 2 hours, we should try to solve there and then.
+
+### Expected to take more than 2 hours
+
+For less critical requests, if we expect delivering the solution to take more than 2 hours, we should allocate the request to the relevant [Customer Roadmap](https://trello.com/b/fXZz8YC3/support-epics), letting the customer know that this course of action was taken. You should have a conversation with the 'Account Manager' for that customer so they can pick up the prioritisation of this work during the next customer support catchup.
+
+Moving the request to the Customer Roadmap should be considered a successful triage resolution, and so the ticket should be closed.
+
+Customers may ask for time estimates for larger pieces of work. This [FAQ answer](process/support/faqs.md) should help.
+
+- [ ] You've [Popped the Why Stack](http://www.mattblodgett.com/2009/01/pop-stack.html) to ensure you understand what commercial goal the customer is trying to solve
+- [ ] You've successfully resolved the request or added to the Customer Roadmap
+- [ ] You can assert that any code change you made has improved the quality of the application
+
+## 2. Record time in Harvest
```

Comment:
And to potentially start at the 0:15 mark, even if the email takes 5 mins to write?